aiohttp/connector.py

```diff
@@ -931,13 +934,7 @@ def _get_ssl_context(self, req: ClientRequest) -> Optional[SSLContext]:
             sslcontext = req.ssl
             if isinstance(sslcontext, ssl.SSLContext):
                 return sslcontext
-            if sslcontext is not None:
-                # not verified or fingerprinted
-                return self._make_ssl_context(False)
-            sslcontext = self._ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
+            if sslcontext is False:
```

Comment:
Can it be just
```suggestion
            if not sslcontext:
```
?

Reply:
Shouldn't it actually be:
```suggestion
            if sslcontext is not True:
```

Reply:
Yeah, I questioned myself there, but this condition should be hit (as the comment suggests) when it is either a non-validating context (i.e. False) or fingerprinted (i.e. Fingerprint()).
```suggestion
            if sslcontext is not True:
```